index.html

```diff
@@ -0,0 +1,278 @@
+              <img
+                src="./assets/images/twitter.png"
+                class="image-advert"
+                alt="twitter"
+              />
+            </a>
+          </li>
+          <li class="list-advert">
+            <a class="link-advert"> English </a>
+          </li>
+          <li class="list-advert">
+            <a class="link-advert"> Mypage </a>
+          </li>
+          <li class="list-advert">
+            <a class="link-advert"> Logout </a>
+          </li>
+        </ul>
+      </nav>
+      <nav class="navbar-primary desktop-only align-section">
+        <ul class="order-logo">
+          <li class="list-logo">
+            <a class="link-logo" href="./index.html">
+              <img
+                src="./assets/images/titleicon.png"
+                class="image-logo"
+                alt="Erriecha Logo"
+              />
+            </a>
+          </li>
+          <li class="list-logo list-close" id="popup-closeid">
+            <a href="#" class="link-logo close-icon">&#215;</a>
+          </li>
+        </ul>
+        <ul class="order-primary">
+          <li class="list-primary">
+            <a href="./about.html" class="link-primary link-about">About</a>
+          </li>
+          <li class="list-primary">
+            <a href="" class="link-primary">Program</a>
+          </li>
+          <li class="list-primary">
+            <a href="" class="link-primary">Join</a>
+          </li>
+          <li class="list-primary">
+            <a href="" class="link-primary">Sponsor</a>
+          </li>
+          <li class="list-primary list-news">
+            <a href="" class="link-primary">News</a>
+          </li>
+          <li class="list-primary list-orgname">
+            <a href="" class="link-primary link-orgname">Erriecha </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="align-section section-primary">
+        <div class="humberger-container mobile-only">
+          <a href="javascrip:void()" class="humbergur-link" id="humbergurid">
+            &#9776;</a
+          >
+        </div>
+        <div class="card-primary">
+          <h1 class="event-motto">
+            <q class="motto-quot">Irreechaa Love,Peace and Unity</q>
+          </h1>
+
+          <h1 class="event-title">CELEBRATING IRREECHAA <br />FESTIVAL 2023</h1>
+          <p class="event-message">
+            A joyful celebration believing in value of openness and sharing.
+            Creating a positive change with people from every corner of the
+            countries is taking place in October in Ethiopian capital Finfinnee.
+          </p>
+          <div class="head-venue">
+            <h3 class="venue-date">2023,10,15(sat)-16(sun)</h3>
+            <p class="venue-location">
+              @Finfinnee, Hora Finfinnee and Bishooftuu, Hora Arsadii
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="align-section program-section">
```

Comment:
- [ ] Great job on the content and the design. :+1: Let's make some modifications to match the design template. Please add some more horizontal and vertical spacing as seen in the design. 🤝️
- [ ] The text `Main Program` and `See the whole program` should be centered. 😎️

**Your design**
![Screenshot from 2023-03-24 11-30-55](https://user-images.githubusercontent.com/17243325/227475411-d87bf493-4265-4cdb-bfcd-9399a1db0228.png)

**Behance design**
![Screenshot from 2023-03-24 11-31-21](https://user-images.githubusercontent.com/17243325/227475516-222dac15-43d1-430e-bf38-35aab1fba661.png)